### PR TITLE
-made strike buttons for each strike

### PR DESCRIPTION
**Pull request by trant017**
-made music button
-removed auto switch to new question after assinging points

### CODE REVIEW

File: src/Controls.js [hunk 1]
```diff
@@ -168,6 +172,41 @@ class Controls extends Component {
...
+      strikeCount,
+      showStrike: true
+    });
+    window.setTimeout(this.hideStrike, 2000);
+  }
+
+  showTwoStrike() {
+    const strikeCount = 2;
+    const dbRef = this.db.ref("/");
+    dbRef.set(this.state);
```
Comment (Mende) [inline src/Controls.js]: `dbRef.set(this.state)` is not needed in any of the strike functions. you could probably dry this out a lot if you kept the single show strike function and simply passed a different count value based off the button that was pressed....

Comment (Mende) [reply src/Controls.js]:
```
showStrikes(strikeCount = 1) {
    const dbRef = this.db.ref("/");
      dbRef.set({
      ...this.state,
      strikeCount,
      showStrike: true
      });
    window.setTimeout(this.hideStrike, 2000);
}
```

Then each buttons would do somehting like this

`
<button onclick={this.showStrikes}>1 Strike</button>
<button onclick={() => {this.showStrikes(2);}}>2 strikes</button>
<button onclick={() => {this.showStrikes(2);}}>3 Strikes </button>`

Comment (Mende) [reply src/Controls.js]: or the better way as noted here
https://stackoverflow.com/questions/29810914/react-js-onclick-cant-pass-value-to-method

**Comment by trant017** [comment]
This PR has been updated

**Pull request merged**